### ISSUE TITLE
only_throw_errors: Ignore files in test/**.dart

### ISSUE THREAD

**Issue by matanlurey** [dart-lang/linter#807]
Doesn't make sense to use a class when you're just in a test.

```dart
// foo_test.dart
void main() {
  // ignore: only_throw_errors
  throw 'Should not lint';
}
```

**Comment by zoechi** [comment]
Or allow globs for individual linter rules in analysis_options.yaml?
